sbi/utils/plot/plot.py

```diff
@@ -246,7 +227,7 @@ def samples_nd(
         samples = [samples]
 
     # Prepare points
-    if points is None: 
+    if points is None:
```

Comment:
This deleted blankspace seems to be the only change in the function. Or am I missing something? Maybe github is also messed up the diff?

Reply:
yes, because this was already adressed in the now merged PR #172

Reply:
to independent solutions ;-)

Reply:
Ah, I see, thanks!